src/utils/templates.ts

```diff
@@ -62,11 +61,10 @@ export async function findConflicts(copyDescriptors: CopyDescriptor[], force = f
       conflicts.forEach(copyDescriptor => fs.unlinkSync(copyDescriptor.fileName));
     } else {
       const listOfFiles = conflicts.map(f => path.basename(f.fileName)).join('", "');
-      cli.error(
+      throw new Error(
         conflicts.length > 1
           ? `Files with the names "${listOfFiles}" already exist. If you want to overwrite them, rerun the command with \`--force\`.`
-          : `A file with the name "${listOfFiles}" already exists. If you want to overwrite it, rerun the command with \`--force\`.`,
-        { exit: 1 }
```

Comment:
In the new version the findConflixts return {exit: 11}  e.g. add-approuter line 43.